### PR TITLE
[Validator] Added missing Turkish (tr) translations

### LINKED ISSUE #51959: Missing translations for Turkish (tr)
Hello,

There are some missing Turkish (tr) translations and we are looking for a **native** speaker to help us out. 

Here is a [short example](https://symfony-translations.nyholm.tech/#pr) of what you need to do. There are 4 rules: 

1. You must be a Turkish (tr) native speaker
2. You must look at the existing translations and follow the same "style" or "tone"
3. You must make your PR to branch 5.4
4. You must use the correct indentation (number of spaces)

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf)

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by shubhalgupta**
| Q             | A
| ------------- | ---
| Branch?       | 5.4 
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Issues        | Fix #51959 
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf [hunk 1]
```diff
@@ -402,6 +402,31 @@
...
+          </trans-unit>
+          <trans-unit id="108">
+           <source>Mixing numbers from different scripts is not allowed.</source>
+           <target>Farklı yazı türlerinden sayıların karıştırılması izin verilmez.</target>
+          </trans-unit>
+          <trans-unit id="109">
+           <source>Using hidden overlay characters is not allowed.</source>
+           <target>Gizli üstü kaplama karakterlerinin kullanılması izin verilmez.</target>
+         </trans-unit>
+
```
Comment (OskarStark) [inline src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf]: blank line should re removed

File: src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf [hunk 2]
```diff
@@ -402,6 +402,31 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Netmask'in değeri {{ min }} ve {{ max }} arasında olmaldır.</target>
             </trans-unit>
+            <trans-unit id="104">
+               <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+               <target>Dosya adı çok uzun. {{ filename_max_length }} karakter veya daha az olmalıdır.|Dosya adı çok uzun. {{ filename_max_length }} karakter veya daha az olmalıdır.</target>
+            </trans-unit>
+            <trans-unit id="105">
+              <source>The password strength is too low. Please use a stronger password.</source>
```
Comment (OskarStark) [inline src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf]: The indentation is wrong, please use the same indentation from line no 406, thanks

File: src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf [hunk 3]
```diff
@@ -402,6 +402,31 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Netmask'in değeri {{ min }} ve {{ max }} arasında olmaldır.</target>
             </trans-unit>
+            <trans-unit id="104">
+               <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+               <target>Dosya adı çok uzun. {{ filename_max_length }} karakter veya daha az olmalıdır.|Dosya adı çok uzun. {{ filename_max_length }} karakter veya daha az olmalıdır.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf]: plural and singular are the same
if this is legit, not need to repeat them

File: src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf [hunk 4]
```diff
@@ -402,6 +402,31 @@
...
+            <source>Using invisible characters is not allowed.</source>
+            <target>Görünmez karakterlerin kullanılması izin verilmez.</target>
+          </trans-unit>
+          <trans-unit id="108">
+           <source>Mixing numbers from different scripts is not allowed.</source>
+           <target>Farklı yazı türlerinden sayıların karıştırılması izin verilmez.</target>
+          </trans-unit>
+          <trans-unit id="109">
+           <source>Using hidden overlay characters is not allowed.</source>
+           <target>Gizli üstü kaplama karakterlerinin kullanılması izin verilmez.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf]: GPT suggest `Gizli karakterlerin kullanılması izin verilmez.` for this one, the current translation is too literal it says.

File: src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf (hunk 3 above)
Comment (stof) [reply src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf]: According to our pluralization rules in TranslatorTrait, Turkish indeed does not have multiple variants

Comment (stof) [reply src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf]: note that we should not have variants at all there, not duplicating the variant.

**Comment by nicolas-grekas** [comment]
Thank you @shubhalgupta.

**Pull request merged**